ENTITY/SetEntityProofs.md

```diff
@@ -8,11 +8,9 @@ ns: ENTITY
 void SET_ENTITY_PROOFS(Entity entity, BOOL bulletProof, BOOL fireProof, BOOL explosionProof, BOOL collisionProof, BOOL meleeProof, BOOL steamProof, BOOL p7, BOOL drownProof);
 ```
 
-```
-Enable / disable each type of damage.  
---------------  
-p7 is to to '1' in am_mp_property_ext/int: entity::set_entity_proofs(uParam0->f_19, true, true, true, true, true, true, 1, true);  
-```
+Makes the specified entity invulnerable to the specified damage type
+
+Setter for [`GET_ENTITY_PROOFS`](_0xBE8CD9BE829BBEBF)
```

Comment:
Needs a preceding `#`.

Setter for [`GET_ENTITY_PROOFS`](#_0xBE8CD9BE829BBEBF)